src/style.css

```diff
@@ -0,0 +1,117 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  src: local("fonts/Avenir-Book.ttf"),
+    url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  src: local("fonts/Avenir-Heavy.ttf"),
+    url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+  box-sizing: border-box;
+  background: none;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
```

Comment:
always set font styles to the `html` tag
in `*` tag only `box-sizing` can be used, anything else is forbidden 
margin and padding you can set to the `body`